instrumentation/opentelemetry-instrumentation-openai/src/opentelemetry/instrumentation/openai/patch.py

```diff
@@ -28,8 +30,10 @@
     set_span_attribute,
     set_event_completion,
     extract_tools_prompt,
+    set_event_prompt,
 )
 from opentelemetry.trace import Tracer
+import json
```

Comment:
looks like you have not setup pre-commit, please see how to do that in CONTRIBUTING.md to save some CI lint failures :)

Reply:
thanks, will do now